sdk/cosmosdb/cosmos/samples-dev/ItemManagement.ts

```diff
@@ -11,29 +11,27 @@ dotenv.config({ path: path.resolve(__dirname, "../sample.env") });
 
 import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError";
 import { readFileSync } from "fs";
-import { CosmosClient } from "../dist-esm";
-const {
-  COSMOS_DATABASE: databaseId,
-  COSMOS_CONTAINER: containerId,
-  COSMOS_ENDPOINT: endpoint,
-  COSMOS_KEY: key
-} = process.env;
+import { CosmosClient } from "@azure/cosmos";
+const key = process.env.COSMOS_KEY || "<cosmos key>";
+const endpoint = process.env.COSMOS_ENDPOINT || "<cosmos endpoint>";
+const containerId = process.env.COSMOS_CONTAINER || "<cosmos container>";
+const dbId = process.env.COSMOS_DATABASE || "<cosmos database>";
 
 logSampleHeader("Item Management");
 
 const itemDefs = JSON.parse(
```

Comment:
Instead of reading the file, you can simply `import` it. Instead of copying it to assets/data in both the JS and TS folders, you can copy it to src/data/families.json in the TypeScript folder and to data/families.json in the JavaScript folder, and then read it like this:

`import families from "./data/families.json";`

Reply:
I hope you are referring the same line above https://github.com/Azure/azure-sdk-for-js/pull/18623#discussion_r749361915.